app/Filament/Resources/MetricResource.php

```diff
@@ -143,15 +143,17 @@ public static function form(Form $form): Form
 
                             ]),
 
-                        Tab::make('Topics and Dimensions')
+                        Tab::make('Dimensions')
```

Comment:
Given we're not using topics at all (It seems clear that they were replaced by "themes" in the ICRAF team's thinking), I think we definitely can remove topics from here and entirely clean them from the database.